src/main.js

```diff
@@ -95,7 +124,29 @@ var promptCredentials = function(callback) {
     } else {
         properties.push(testdirConfig);
     }
+    
+    //Pull requests flag
+    if (argv.pullrequest) {
+        gPullRequestFlag = argv.pullrequest;
+    } else {
+        properties.push(pullrequestConfig);
+    }
 
+    //Filter by label(s) 
+    if (argv.filterissues) {
+        gFilterIssueLabels = argv.filterissues;
+        gFilterIssueLabels = gFilterIssueLabels.split(", ");
+    } else {
+        properties.push(filterissuesConfig);
+    }
+
+    //Filter by keyword search 
+    if (argv.keywords) {
+        gKeywords = argv.keywords;
+        gKeywords = gKeywords.split(", ");
```

Comment:
split by only commas, not on any whitespace. then remove the whitespace from each token. let me know if you dont understand what i mean.